### ISSUE TITLE
Just reporting: 2023 T962 - Upgrade successful

### ISSUE THREAD

**Issue by ChitlangeSahas** [UnifiedEngineering/T-962-improvements#243]
Thank you for this upgrade, I just wanted to report that this still works on a 2023 T962. I see a bunch of questions around whether this is working well for "newer" builds. 

It is for me. 

Thank you again :)

**Comment by nophead** [comment]
Yes I applied the mods to a T962  that I purchased from Elektor a year ago and hadn't used until now. Everything seemed the same, same PCB, masking tape insulation, etc. The only small improvement was the paint around the earth connection had been masked off, so it did actually make contact. It wouldn't pass any regulations because you are not supposed to re-use a mechanical fastening as an earth.  It should be a separate screw that just attaches the earth. However it is good enough for me and the top case is earthed by the countersunk screws above the drawer as the countersink is free of paint. Again far from ideal but good enough for me.

I glued the DS18B20+ cold junction sensor to the terminal block and the PCB using JB-Weld epoxy that I think is reasonably thermally conductive.
